tilt-settings.yaml

```
@@ -0,0 +1,49 @@
+
+helm_installation_name: gloo-oss
+helm_values_file: ./test/kube2e/helm/artifacts/helm.yaml
+helm_installation_namespace: gloo-system
+
+enabled_providers:
+  - gloo
+  - gateway-proxy
+
+providers :
+  gloo:
+    context: _output/projects/gloo
+    image: quay.io/solo-io/gloo
+    live_reload_deps:
+      - projects/gloo
+    label: gloo
+    build_binary: make -B gloo
+    binary_name: gloo-linux-arm64
```

Not all devs will be using this binary
If there's a way to dynamically determine which should be used, that would be great, though I'm not sure if that's reasonable

As an alternative we can add a commented line that calls out that some devs may need to use the AMD binary
```suggestion
    binary_name: gloo-linux-arm64
    # binary_name: gloo-linux-amd64 # devs on AMD machines will need to use this binary instead
```

---

Thanks, I've updated the docs as well as made provision for the architecture